tools/econfgen/econfgen.yml.example

```diff
@@ -0,0 +1,44 @@
+---
+default:
+  # set up a backend
+  backend:
+    name: git-gateway
+  local_backend: true
+  # the jobs section is as follows:
+  # jobs:
+  #   ROLE:
+  #   - ['Long Job Name', 'lgn']
+  jobs:
+    tanks:
+    - ['Paladin', 'pld']
+    - ['Warrior', 'war']
+    - ['Dark Knight', 'drk']
+    - ['Gunbreaker', 'gbn']
+    healers:
+    - ['Scholar', 'sch']
+    - ['White Mage', 'whm']
+    - ['Astrologian', 'ast']
+    - ['Sage', 'sge']
+    melee:
+    - ['Monk', 'mnk']
+    - ['Dragoon', 'drg']
+    - ['Ninja', 'nin']
+    - ['Samurai', 'sam']
+    - ['Reaper', 'rpr']
+    ranged:
+    - ['Bard', 'brd']
+    - ['Machinst', 'mch']
```

Comment:
Spelling error here for MCH, should be Machinist based on current generated yaml, catches if you just copy the example out.